kevm-pyk/src/kevm_pyk/kproj/evm-semantics/evm.md

```diff
@@ -1438,7 +1438,8 @@ The various `CALL*` (and other inter-contract control flow) operations will be d
            ...
          </account>
 
-    rule <k> #call ACCTFROM ACCTTO ACCTCODE VALUE APPVALUE ARGS STATIC
+    rule [call.false]:
+         <k> #call ACCTFROM ACCTTO ACCTCODE VALUE APPVALUE ARGS STATIC
```

Comment:
Should we label the corresponding other one `call.true` then?

Reply:
Yes, I don't think the `call.true` will be used in the short future, but I added it for consistency together with other labels.